src/sentry/static/sentry/app/views/performance/landing.tsx

```diff
@@ -176,18 +177,38 @@ class PerformanceLanding extends React.Component<Props, State> {
       ...location.query,
     };
 
+    const query = decodeScalar(location.query.query) || '';
+    const conditions = tokenizeSearch(query);
+
     // This is a temporary change for trends to test adding a default count to increase relevancy
     if (viewKey === FilterViews.TRENDS) {
-      if (!newQuery.query) {
-        newQuery.query =
-          'count():>1000 transaction.duration:>0 p50():>0 avg(transaction.duration):>0';
+      const hasStartAndEnd = newQuery.start && newQuery.end;
+      if (!newQuery.statsPeriod && !hasStartAndEnd) {
+        newQuery.statsPeriod = DEFAULT_TRENDS_STATS_PERIOD;
+      }
+      if (!query) {
+        conditions.setTag('count()', ['>1000']);
+        conditions.setTag('transaction.duration', ['>0']);
       }
-      if (!newQuery.query.includes('count()')) {
-        newQuery.query += 'count():>1000';
+      if (!query.includes('count()')) {
+        conditions.setTag('count()', ['>1000']);
       }
-      if (!newQuery.query.includes('transaction.duration')) {
-        newQuery.query += ' transaction.duration:>0';
+      if (!conditions.getTags('transaction.duration')) {
```

Comment:
It looks like it might be possible that `getTags` returns an empty array, seems next to impossible, but would be good to address it here on the off chance it does happen,

Reply:
hmm thinking about it I'm going to add a function to QueryResults instead to do tag existence check since it's logic needs a check for length perhaps as well.